tests/e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -314,4 +314,3 @@ async function checkJavaPathCompletion() {
         await editor.waitTabWithSavedStatus(classPathFilename);
     }
 }
-
```

Comment:
Looks like redundant change.

Reply:
In fact, there are two empty lines in the end.
I think, it's redundant, so I deleted one empty line.

Reply:
It's a common practice to have at least one empty line at the end of script https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file